OWNERS_ALIASES

```diff
@@ -12,6 +12,9 @@ aliases:
   kubebuilder-approvers:
     - camilamacedo86
     - vincepri
+    - adirio
+    - jmrodri
+    - varshaprasad96
```

Comment:
Hi @varshaprasad96 the adirio we should probably move to kubebuilder-emeritus-approvers: he is no longer helping on this one. 

Otherwise, all fine